src/emulator/functionsEmulator.ts

```diff
@@ -808,6 +809,21 @@ export class FunctionsEmulator implements EmulatorInstance {
       }
     }
 
+    // Yarn 2 has a new feature called PnP (Plug N Play) which aims to completely take over
+    // module resolution. This feature is mostly incompatible with CF3 (prod or emulated) so
+    // if we detect it we should warn the developer.
+    // See: https://classic.yarnpkg.com/en/docs/pnp/
+    const pnpPath = path.join(frb.cwd, ".pnp.js");
```

Comment:
@samtstern The file is called `.pnp.cjs` for new versions of yarn2. Would be good to include the new name, I didn't see the warning until I tried the deployment.

Reply:
@barbogast would you like to send a PR to fix this?  I guess we should check for both.